src/test/java/paimon/tasklist/TaskListTest.java

```diff
@@ -0,0 +1,63 @@
+package paimon.tasklist;  //same package as the class being tested
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import paimon.items.Deadline;
+import paimon.items.Todo;
+import paimon.items.Event;
+
+public class TaskListTest {
+    @Test
+    public void addTest(){
```

Comment:
Does it make more sense to have addTest() or testAdd() as the method name here?
```suggestion
    public void testAdd(){
```